Modules/System/Translation/src/Translation.Codeunit.al

```diff
@@ -109,6 +109,34 @@ codeunit 3711 Translation
         TranslationImplementation.Delete(RecVariant, FieldId);
     end;
 
+    /// <summary>
+    /// Copies the translation for a field from on record to another record on a persisted (non temporary) record.
+    /// </summary>
+    /// <param name="FromRecVariant">The record from which the translations are copied.</param>
+    /// <param name="ToRecVariant">The record to which the translations are copied.</param>
+    /// <error>If the RecVariant parameter is the type Record, and it is temporary.</error>
+    /// <error>If the RecVariant parameter is the type Record, and the table number is 0.</error>
+    /// <error>If the FromRecVariant parameter is the type Record, the ToRecVariant parameter is the type Record and they are differnt.</error> 
+    procedure Copy(FromRecVariant: Variant; ToRecVariant: Variant)
+    begin
+        TranslationImplementation.Copy(FromRecVariant, ToRecVariant);
+    end;
+
```

Comment:
Your implementation looks alright, but before we can start processing this PR, you need to add some tests which cover your newly implemented code - else this PR won't pass the code coverage requirements.

Reply:
@JesperSchulz , I tried to add a test, but since the pipeline is still failing (`Your program license does not allow you to publish 'Word Templates Test Table'.`), I'm having problmes to ensure that my tests are correct.

Reply:
The pipeline is no longer failing! Well, it is, but that's because one of your tests is failing :-)

Error:
  Assert.AreEqual failed. Expected:<Translation 1> (Text). Actual:<> (Text). The translation should have been copied.
Call Stack:
  Library Assert(CodeUnit 130002).AreEqual - Library Assert by Microsoft
  Translation Tests(CodeUnit 137121).TestCopyTranslations - System Application Test by Microsoft
  Test Runner - Mgt(CodeUnit 130454).RunTests - Test Runner by Microsoft
  Test Runner - Isol. Codeunit(CodeUnit 130450).OnRun - Test Runner by Microsoft
  Test Suite Mgt.(CodeUnit 130456).RunTests - Test Runner by Microsoft
  Test Suite Mgt.(CodeUnit 130456).RunSelectedTests - Test Runner by Microsoft
  Test Suite Mgt.(CodeUnit 130456).RunNextTest - Test Runner by Microsoft
  Command Line Test Tool(Page 130455).RunNextTest - OnAction - Test Runner by Microsoft

Reply:
This error is now resolved.
I tried to add another test, to ensure my error with two different tables. For this I added a new table with ID `table 137122 "Translation Test Table Two"` that isn't contained in the current license.
Do you prefer some other form of testing or how should I resolve this?

Reply:
Unfortunately, we cannot add new test tables in minor releases. Tables and pages for tests are only updated on major releases - at least for now. It's not ideal, but those are currently the rules of the game. If you can somehow get away with creating tests without pages and tables, it would be ideal. If not, I can add your tests to our upcoming major release, while we'll disable the tests for this repo (minor release).

Reply:
Okay. That's not ideal. I thought about abusing a system table, like Media instead of creating a new Translation Test Table.
I'm not sure what would be the best solution.
Only one test needs this additional table.
I would prefer to uncomment this test and the test table for now, since I'm unsure about abusing the System Table Media for this test...

Reply:
That's perfectly alright. I'll comment the test and table in, when forward porting your changes to our major release branch. Btw. "abusing" a table for test is also perfectly alright :-)